jetbrains-gateway/main.tf

```diff
@@ -58,37 +58,27 @@ locals {
     "GO" = {
       icon  = "/icon/goland.svg",
       name  = "GoLand",
-      value = jsonencode(["GO", "232.9921.53", "https://download.jetbrains.com/go/goland-2023.2.2.tar.gz"])
+      value = jsonencode(["GO", "232.10203.20", "https://download.jetbrains.com/go/goland-2023.2.4.tar.gz"])
```

Comment:
I don't think we can just redefine these options.  Existing workspaces that were built with the old options have the old values stored as parameters.  Then, when trying to rebuild the workspace, it tries to look up the display name and icon using the old value and `index` and this is what is failing.

Reply:
But should not this download the newer module and use the new values? I am confused about why it would use the old values.

Reply:
Yeah, I think that without a new template version, Coder doesn't spot that the parameter value from the old build is potentially outdated, so it doesn't trigger reselecting a valid parameter value.

Reply:
Could you approve it for now? We can also force update the version

Reply:
> We can also force update the version

So this will break other deployments that use our module?

Reply:
yes it will.

Reply:
How urgent is this update?  

The breakage is pretty confusing, and if your workspace is running you can't update because that runs a stop, which is broken.

I really think we should wait until we have a solution that doesn't break us or others.

Reply:
It is not urgent and we can wait for a fix.